warning: unused variable 'stuff_to_do'

**warning: unused variable 'stuff_to_do'** 

```
.pio/libdeps/esp32dev/FastLED/src/FastLED.h:14:21: note: #pragma message: FastLED version 3.004.000
 #    pragma message "FastLED version 3.004.000"

.pio/libdeps/esp32dev/FastLED/src/platforms/esp/32/clockless_rmt_esp32.cpp: In static member function 'static void ESP32RMTController::interruptHandler(void*)':
.pio/libdeps/esp32dev/FastLED/src/platforms/esp/32/clockless_rmt_esp32.cpp:318:10: warning: unused variable 'stuff_to_do' [-Wunused-variable]
     bool stuff_to_do = false;```

--- a/src/platforms/esp/32/clockless_rmt_esp32.cpp
+++ b/src/platforms/esp/32/clockless_rmt_esp32.cpp
@@ -326,7 +326,6 @@ void IRAM_ATTR ESP32RMTController::interruptHandler(void *arg)
     uint32_t intr_st = RMT.int_st.val;
     uint8_t channel;
 
-    bool stuff_to_do = false;
     for (channel = 0; channel < gMaxChannel; channel++) {
         int tx_done_bit = channel * 3;
         int tx_next_bit = channel + 24;

